src/components/drop-down-options/drop-down-options.scss

```diff
@@ -15,10 +15,17 @@
     min-width: fit-content;
   }
 
-  .ellipsis-overflow {
+  &.ellipsis-overflow {
     @include ellipsis-overflow;
   }
 
+  .c-options-grid {
+    display: inline-grid;
```

Comment:
actually this should probably have been `display: grid` since we don't need it inline, but it won't make a difference since we don't actually (and will likely never) have text around this grid